configuration/security.md

```diff
@@ -208,30 +263,43 @@ The status command lists all current settings for the firewall:
 
 ```bash
 sudo ufw status
+sudo ufw status verbose
+sudo ufw status numbered
 ```
 
 The rules can be quite complicated, allowing specific IP addresses to be blocked, specifying in which direction traffic is allowed, or limiting the number of attempts to connect, for example to help defeat a Denial of Service (DoS) attack. You can also specify the device rules are to be applied to (e.g. eth0, wlan0). Please refer to the `ufw` man page (`man ufw`) for full details, but here are some examples of more sophisticated commands.
 
 Limit login attempts on ssh port using tcp: this denies connection if an IP address has attempted to connect six or more times in the last 30 seconds:
```

Comment:
It seems the example following this text has disappeared?

Reply:
What example are you referring to? I've checked and everything I've added is still there as far as I can see.

Reply:
It seems you've accidentally removed the `sudo ufw limit ssh/tcp` line?

Reply:
Oh, I got rid of that because I didn't see it as necessary since there are already examples for that. Should I put it back?

Reply:
That's up to you - but if you leave it out then I think you need to remove the "Limit login attempts on ssh port using tcp: ..." bit too?